docs/instructions_nuimages.md

```diff
@@ -12,13 +12,23 @@
 
 # Introduction
 In nuImages, we annotate objects with 2d boxes, instance masks and 2d segmentation masks. All the labels and attributes from nuScenes are carried over into nuImages.
-We have also added more attributes in nuImages. For segmentation, we have included ["stuff" (background) classes](#surfaces).
+We have also [added more attributes](#attributes) in nuImages. For segmentation, we have included ["stuff" (background) classes](#surfaces).
 
 
 
 # Objects
 nuImages contains the same [object classes](https://github.com/nutonomy/nuscenes-devkit/tree/master/docs/instructions_nuscenes.md#labels),
-while the [attributes](TODO link to website) are a superset of the [attributes in nuScenes.](https://github.com/nutonomy/nuscenes-devkit/tree/master/docs/instructions_nuscenes.md#attributes).
+while the [attributes](#attributes) are a superset of the [attributes in nuScenes.](https://github.com/nutonomy/nuscenes-devkit/tree/master/docs/instructions_nuscenes.md#attributes).
+
+## Attributes
```

Comment:
Let's move these after "Instance Segmentation" and add them in the table of contents.
Also explain that each object comes with a box, a mask and a set of attributes.